src/backend/launcher.ts

```diff
@@ -862,13 +863,32 @@ async function callRunner(
   runner: RunnerProps,
   options?: CallRunnerOptions
 ): Promise<ExecResult> {
-  const fullRunnerPath = join(runner.dir, runner.bin)
   const appName = commandParts[commandParts.findIndex(() => 'launch') + 1]
 
   // Necessary to get rid of possible undefined or null entries, else
   // TypeError is triggered
   commandParts = commandParts.filter(Boolean)
 
+  let bin = runner.bin
+  let fullRunnerPath = join(runner.dir, bin)
+
```

Comment:
Since we change dir when executing the `spawn` command bellow, changing the `fullRunnerPath` to be only the `runner.bin` fixes the issue I am having with all legendary commands failing due to spaces in my user name.